//
// Copyright 2017 Asylo authors
//
// Licensed under the Apache License, Version 2.0 (the "License");
// you may not use this file except in compliance with the License.
// You may obtain a copy of the License at
//
//     http://www.apache.org/licenses/LICENSE-2.0
//
// Unless required by applicable law or agreed to in writing, software
// distributed under the License is distributed on an "AS IS" BASIS,
// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
// See the License for the specific language governing permissions and
// limitations under the License.
//

syntax = "proto2";

package asylo;

import "asylo/enclave.proto";

message EnclaveApiTest {
  optional string test_string = 1;
  optional int32 test_int = 2;
  repeated string test_repeated = 3;
}

extend EnclaveInput {
  optional EnclaveApiTest enclave_api_test_input = 157851293;
}

extend EnclaveOutput {
  optional EnclaveApiTest enclave_api_test_output = 157239766;
}
